mahta/TS/TS basics/basics.ts

```diff
@@ -0,0 +1,73 @@
+function add(a: number, b: number): number {
+    return a + b;
+}
+
+function flipACoin(): string {
+    const random: number = Math.random();
+    if (random > 0.5) {
+        return "tail";
```

Comment:
```suggestion
        return "tail" as const;
```

Reply:
please read this article about `as const`
https://www.totaltypescript.com/concepts/as-const
https://stackoverflow.com/questions/66993264/what-does-the-as-const-mean-in-typescript-and-what-is-its-use-case